HelpSource/Classes/MIDIFunc.schelp

```diff
@@ -6,8 +6,9 @@ related:: Guides/MIDI, Classes/MIDIdef
 DESCRIPTION::
 MIDIFunc (and its subclass link::Classes/MIDIdef::) registers one or more functions to respond to an incoming MIDI message. Many of its methods are inherited from its superclass link::Classes/AbstractResponderFunc::.
 
-note:: MIDIFunc and link::Classes/MIDIdef:: aim to improve upon the MIDIresponder classes by being faster, easier to use, and providing support for all MIDI message types. They were made with the intention of creating a more convenient, logical and consistent interface, which shares a common design with link::Classes/OSCFunc:: and link::Classes/OSCdef::. Note that unlike the older classes, MIDIFuncs are removed on Cmd-. by default. This can be overridden using either of the fix or permanent methods.::
+subsection:: Important note on persistence
 
+MIDIFunc (and its subclass link::Classes/MIDIdef:: like all other AbstractResponderFuncs) are removed on pressing Cmd/ctrl-. To fix them, call link::Classes/AbstractResponderFunc#-fix#fix::.
```

Comment:
Instead of `To fix them...`, which might suggest they can be fixed after pressing Cmd-., I'd prefer something like `To override this behavior, use the permanent method`, with `permanent` linking to the documentation entry. Also I'd prefer that over `fix` since fix is a synonym for permanent.

Reply:
Thanks! Should be fixed now.